src/lib/button/button.html

```diff
@@ -1,6 +1,7 @@
 <span class="mat-button-wrapper"><ng-content></ng-content></span>
 <div md-ripple *ngIf="!_isRippleDisabled()" class="mat-button-ripple"
-    [class.mat-button-ripple-round]="_isRoundButton"
-    [mdRippleTrigger]="_getHostElement()"></div>
+     [class.mat-button-ripple-round]="_isRoundButton || _isIconButton"
```

Comment:
The `_isIconButton` check can be removed. `_isRoundButton` also includes the icon buttons.